front_camera/front_camera/get_correction_angle.py

```diff
@@ -39,13 +42,13 @@ def calculate_correction_angle(self, image_path):
         image_center_x = image.shape[1] // 2
         # image_center_y = image.shape[0] // 2
 
-        # 黄色い長方形を検出
-        yellow_rect = self.yellow_rectangle_detector.detect_yellow_rectangle(image)
-        if yellow_rect is None:
-            raise ValueError("黄色い長方形が検出されませんでした")
+        # 対象色の長方形を検出
+        target_color_rect = self.rectangle_detector.detect_rectangle(image)
+        if target_color_rect is None:
+            raise ValueError(f"{self.rectangle_detector.color}色の長方形が検出されませんでした")
```

Comment:
カスタムエラークラスを使う場合、以下2つの変更も必要。
```suggestion
            raise RectangleDetectionError(f"{self.rectangle_detector.color}色の長方形が検出されませんでした")
```

少し上の行のraiseの部分を、
``` 
raise ImageLoadError(f"画像を読み込めませんでした: {image_path}")
```
に変更する。